src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -516,6 +516,82 @@ default Path resolve(String other) {
         return resolve(getFileSystem().getPath(other));
     }
 
+    /**
+     * Resolves a path, or a sequence of paths against this path.
```

Comment:
I think we'll need to come up with a better first line because the method doesn't resolve the sequence of paths against this path, it instead resolves the first path against this path, then iteratively resolves.

I think the second paragraph will need to include something about empty elements, as we have in the existing resolve spec.